### PR TITLE
Feature/update to polkadot v0 9 26

### PR DESCRIPTION
**Pull request by MRamanenkau**

### CODE REVIEW

File: runtime/cere/Cargo.toml
Comment (AndreiNavoichyk) [inline runtime/cere/Cargo.toml:1]: Not found changes:
1. `tiny_kessak`

**Review by AndreiNavoichyk — COMMENTED** [review]
@MRamanenkau 
Please, apply all comments from `cere` runtime to `cere-dev`.

File: runtime/cere-dev/Cargo.toml [hunk 1]
```diff
@@ -14,7 +14,7 @@ targets = ["x86_64-unknown-linux-gnu"]
 [dependencies]
 
 # third-party dependencies
-codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = [
+codec = { package = "parity-scale-codec", version = "3.1.5", default-features = false, features = [
```
[the remark this answers is not in the document]
Comment (MRamanenkau) [reply runtime/cere-dev/Cargo.toml:17]: Updated

File: runtime/cere/src/lib.rs
[the remark this answers is not in the document]
Comment (MRamanenkau) [reply runtime/cere/src/lib.rs:1]: the line from Polkadot doesn't work for us. I've taken it from [Substrate](https://github.com/paritytech/substrate/blob/polkadot-v0.9.26/bin/node/runtime/src/lib.rs#L1590)

File: runtime/cere/src/lib.rs [hunk 2]
```diff
@@ -981,6 +985,7 @@ impl pallet_contracts::Config for Runtime {
 	type ContractAccessWeight = pallet_contracts::DefaultContractAccessWeight<RuntimeBlockWeights>;
 	type MaxCodeLen = ConstU32<{ 128 * 1024 }>;
 	type RelaxedMaxCodeLen = ConstU32<{ 256 * 1024 }>;
+	type MaxStorageKeyLen = ConstU32<128>;
```
[the remark this answers is not in the document]
Comment (MRamanenkau) [reply runtime/cere/src/lib.rs:988]: Polkadot's runtime doesn't have pallet-contracts as a dependency.
This line is taken from [Substrate](https://github.com/paritytech/substrate/blob/polkadot-v0.9.26/bin/node/runtime/src/lib.rs#LL1154C7-L1154C23)

File: runtime/cere/src/lib.rs [hunk 3]
```diff
@@ -1580,7 +1585,7 @@ impl_runtime_apis! {
 
 		fn get_storage(
 			address: AccountId,
-			key: [u8; 32],
+			key: Vec<u8>,
```
[the remark this answers is not in the document]
Comment (MRamanenkau) [reply runtime/cere/src/lib.rs:1588]: Polkadot runtime doesn't have this method but Cere does as it was forked from Substrate. 
The line is taken from the [Substrate](https://github.com/paritytech/substrate/blob/master/bin/node/runtime/src/lib.rs#L2336) repo.

File: runtime/cere/Cargo.toml
Comment (MRamanenkau) [reply runtime/cere/Cargo.toml:1]: `keccak` is a hashing algorithm and this lib is not used inside of Polkadot runtime.
 Looks like Parity team added it as a dependency by mistake.

File: runtime/cere/src/lib.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (AndreiNavoichyk) [reply runtime/cere/src/lib.rs:988]: Please, test it for our SCs.

**Review by AndreiNavoichyk — COMMENTED** [review]
Please, increase `spec_version` in cere and cere_dev runtimes.

File: runtime/cere/src/lib.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (MRamanenkau) [reply runtime/cere/src/lib.rs:988]: Tested

**Comment by MRamanenkau** [comment]
> Please, increase `spec_version` in cere and cere_dev runtimes.

Done

**Comment by MRamanenkau** [comment]
> @MRamanenkau Please, apply all comments from `cere` runtime to `cere-dev`.

Done

File: runtime/cere-dev/src/lib.rs [hunk 4]
```diff
@@ -126,7 +128,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 43000,
+	spec_version: 42002,
```
Comment (AndreiNavoichyk) [inline runtime/cere-dev/src/lib.rs:131]: `spec_version` in `dev` is `43000`. Here should be `43001`. Please, update

Comment (MRamanenkau) [reply runtime/cere-dev/src/lib.rs]: Updated

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**